sqlmesh/core/config/connection.py

```diff
@@ -381,14 +382,20 @@ def _static_connection_kwargs(self) -> t.Dict[str, t.Any]:
         """kwargs that are for execution config only"""
         from sqlmesh import __version__
 
-        custom_user_agent_config = {"custom_user_agent": f"SQLMesh/{__version__}"}
         connection_str = "md:"
+
+        query_params = {}
+        query_params["custom_user_agent"] = f"SQLMesh/{__version__}"
```

Comment:
Are you sure that the `custom_user_agent` can be passed like this? Judging by https://github.com/duckdb/duckdb/blob/main/tools/pythonpkg/src/pyconnection.cpp#L2102-L2105 I don't see it being extracted from the URI

Reply:
https://motherduck.com/docs/integrations/how-to-integrate/

db, err := sql.Open("duckdb", "md:my_database?custom_user_agent=INTEGRATION_NAME")